backstop_data/engine_scripts/casper/user.js

```diff
@@ -10,6 +10,12 @@ module.exports = function (casper, scenario) {
   } else if (scenario.user === 'admin') {
     var email = system.env.DM_PRODUCTION_ADMIN_USER_EMAIL;
     var password = system.env.DM_PRODUCTION_ADMIN_USER_PASSWORD;
+  } else if (scenario.user === 'category') {
+    var email = system.env.DM_PRODUCTION_ADMIN_CCS_CATEGORY_USER_EMAIL;
+    var password = system.env.DM_PRODUCTION_ADMIN_CCS_CATEGORY_USER_PASSWORD;
+  } else if (scenario.user === 'sourcing') {
+    var email = system.env.DM_PRODUCTION_ADMIN_CCS_SOURCING_USER_EMAIL;
+    var password = system.env.DM_PRODUCTION_ADMIN_CCS_SOURCING_USER_PASSWORD;
```

Comment:
We'll need to make sure these environment variables are passed into the container when it's run through Jenkins. At the moment in the (presumably) corresponding PRs on jenkins https://github.com/alphagov/digitalmarketplace-jenkins/pull/56 and credentials https://github.com/alphagov/digitalmarketplace-credentials/pull/44, we're passing in variables called `DM_PRODUCTION_ADMIN_MANAGER_USER_*` and `DM_PRODUCTION_ADMIN_FRAMEWORK_MANAGER_USER_*` which don't tie up to what we're expecting here. So we'll need to reconcile that first.

Reply:
(Jenkins job needs to be updated to provide the required env vars before this can be merged)

Reply:
Ben's PRs are unrelated to this one - those are new roles that with haven't added any tests for here. The CCS_SOURCING and CCS_CATEGORY environment bars already exist at least somewhere in Jenkins so I thought this might just work. We can look at it together this morning maybe when I get in if you have a sec? On my phone at the moment so can't easily check where I found these vars.

Reply:
Makes sense. Sorry for thinking the two were inter-related.

The change should be fairly simple for adding the sourcing/category credentials to the job in Jenkins. Just need to update it to declare and pass through matching variables following the pattern provided in the job at https://github.com/alphagov/digitalmarketplace-jenkins/blob/master/job_definitions/visual_regression.yml#L69 and https://github.com/alphagov/digitalmarketplace-jenkins/blob/master/job_definitions/visual_regression.yml#L82